Makefile

```diff
@@ -276,7 +276,7 @@ ifneq (,$(QEMUVIRTIOFSCMD))
     # qemu-specific options (all should be suffixed by "_QEMU")
     DEFBLOCKSTORAGEDRIVER_QEMU_VIRTIOFS := virtio-fs
     DEFNETWORKMODEL_QEMU := tcfilter
-    KERNELNAMEVIRTIOFS = $(call MAKE_KERNEL_VIRTIOFS_NAME,$(KERNELTYPE))
+    KERNELNAMEVIRTIOFS = $(call MAKE_KERNEL_NAME,$(KERNELTYPE))
```

Comment:
Not required for this PR, but we could do with some underscores to make some of these long variable names more readable ;)